Tribler/Core/Modules/restapi/my_channel_endpoint.py

```diff
@@ -90,3 +93,30 @@ def render_GET(self, request):
         for torrent in torrents:
             torrent_list.append({'name': torrent[0], 'infohash': torrent[1].encode('hex'), 'added': torrent[2]})
         return json.dumps({'torrents': torrent_list})
+
+
+class MyChannelRssFeedsEndpoint(MyChannelBaseEndpoint):
+    """
+    Return the RSS feeds in your channel.
+
+    Example response:
+    {
+        "rssfeeds": [{
+            "url": "http://rssprovider.com/feed.xml",
+        }, ...]
+    }
+    """
+
+    def render_GET(self, request):
+        my_channel_id = self.channel_db_handler.getMyChannelId()
+        channel_obj = self.session.lm.channel_manager.get_my_channel(my_channel_id)
+        if my_channel_id is None or channel_obj is None:
+            return MyChannelBaseEndpoint.return_404(request)
+
+        rss_list = channel_obj.get_rss_feed_url_list()
+        request.setHeader('Content-Type', 'text/json')
+        feeds_list = []
+        for rss_item in rss_list:
+            feeds_list.append({'url': rss_item})
```

Comment:
@devos50 @whirm I think this is right place. The non-GUI channel and RSS code are now in `Tribler/Core/Modules/channel`.